turbo.json

```diff
@@ -95,6 +95,15 @@
     "clean": {
       "cache": false
     },
+    "e2e": {
+      "cache": false,
+      "persistent": true,
```

Comment:
As far as I understand, this task is not persistent, since it can terminate after the end of the job (unlike `dev` for example)